nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs) ipfs;
+
+  cfg = config.services.ipfs;
+
+
+  ipfsFlags = ''${if cfg.autoMigrate then "--migrate" else ""}
+                ${if cfg.enableGC    then "--enable-gc" else ""}
+                ${toString cfg.extraFlags}'';
```

Comment:
I just removed the newlines inside the string. Once more options for `ipfs` arise (like listeners, custom config etc.) `ipfsFlags` need to be refactored anyway.